### DISCUSSION TITLE
최상위 폴더 구조에 대한 의견입니다.

### DISCUSSION THREAD

**Discussion by sokojh** [heading1/WYLSBingsu#20] [Polls]
# 문제

최상위 폴더의 이름을 어떻게 할지에 대한 의견이 나와서 투표를 올립니다~!

@Circlewee 노준님이 다른 프로젝트를 참고했는데  최상위 폴더가 

backend, frontend 로 나누어져 있다고 하셔서  저희도 이렇게 진행하면 좋을 것 같다는 의견이십니다.

저같은 경우도 저희가 참고했던 우아한 테크코스 https://github.com/woowa-techcamp-2021 의 프로젝트들의 폴더구조가 전부 client, server로 나누어져 있어서 저희도 그냥 이렇게 진행하는 줄알고 이슈와 함께 폴더 셋팅을 진행했었습니다.

따로 폴더 구조에대해 의논한적이 없는데 그런줄알고 넘어간건 저의 성급한 소통 오류였던 것 같습니다. ㅎ..😅

그래서 투표를 올립니다 !

**Comment by Circlewee** [comment]
어떤 방식으로든 저희가 알아보기 좋은 방법이면 무엇이든 괜찮다고 생각합니다!
의견 반영해서 투표올려주셔서 감사합니다 지호님😀

**Reply by sokojh** [reply]
아닙니당. 제가 놓친부분을 상기시켜주셔서 감사합니다. 👍

**Comment by sokojh** [comment]
투표가 동일하게 나온 이유로 사다리 타기 진행하겠습니다!

1번 frontend, backend 폴더
2번 client, server 폴더
사다리 타기 결과입니다. 
아니 한줄 나오는거 ㅋㅋㅋㅋㅋ 🤣
최상이 폴더는 2번으로 진행하면 될 것 같습니다 ~ 

https://user-images.githubusercontent.com/83265848/173306247-49d744d1-0b7f-4cff-a73c-686b603392fa.mov

**Comment by jinux127** [comment]
생각해보니 프론트엔드, 백엔드라고 나누면 경계가 모호해질수도 있으니 client, server로 나누는게 좋다는 생각이 드네요~!

**Reply by sokojh** [reply]
오 생각해보니 그런 점도 있었군요. 
프로젝트 자체 기능점에 분류를 해야 좋을 것 같긴해요.
